components/FeatureMoblieView.tsx

```diff
@@ -124,7 +124,7 @@ export default function FeaturesMobileView() {
                   </div>
 
                 </div>
-                <a
+                <div
```

Comment:
Hey @vibhaw1904  I think using a Link tag instead of a div tag would be more appropriate here.

Reply:
@Hermione2408  but using link tag ceating a issue that we can't  use link tag inside the link tag ,and in the upper section div was used so i think using div is good idea.

Reply:
Got it, since the description of the issue was not clearly mentioned so I got confused whether you were trying to replacing a tag with Link.